can-auth/vatican.c

```diff
@@ -297,15 +298,15 @@ int VULCAN_FUNC vulcan_recv(ican_t *ican, uint16_t *id, uint8_t *buf, int block)
             recv_len = vatican_receive(ican, &id_recv, mac_recv.bytes, /*block=*/1);
             iat_nonce = decode_iat(iat_timings[interrupt_index] - mac_timing);
             fail = (id_recv != *id + 1) || (recv_len != CAN_PAYLOAD_SIZE) ||
-                (mac_me.quad != mac_recv.quad);
+                (! compare_ct_time((uint8_t *)&mac_me.quad, (uint8_t *)&mac_recv.quad, 4));
```

Comment:
hm this should be 8(?) 64-bits MACs 64/8=8

Generally hard coding is dangerous! Especially for a crucial constant like mac length. Better to use CAN_PAYLOAD_SIZE

Reply:
also need to update leia
https://github.com/sancus-pma/vulcan/blob/fa1126c5a48a785d6dabff5c1dc9338acf43a455/can-auth/leia.c#L346